### DISCUSSION TITLE
(pie) charts

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1681] [users: ideas for new features]
Now that we have a pie chart in the preprocessing ceramics module under collected material everybody has seen how that works and what it looks like.

What I am still waiting for: Where else would such a chart (can be any chart type) be useful in the recording itself?
What about, for instance, a pie chart about what materials are found within a locus?
Or a unit?

@lbestock 
@luizaogs

**Comment by luizaogs** [comment]
First thoughts as I breakfast (after waking up at 2:30am and being unable to fall asleep again, so if this makes no sense I apologize). 

I remember charts/graphs being one of the things I liked the most about the Amara West database, so maybe going through those screenshots in the wiki would be useful. There it is used in analysis rather than recording, though, and it might be the case that the suggestions below are also more useful for the former rather than the latter.

But materials would be useful, I think, and in addition to materials from a specific unit/locus it might be good to have something that shows materials from specific units found across a site? 

Period is also something that strikes me as helpful, but of course that depends on the level of chronological resolution at the site in the first place.
